releases/release-6.5.4.md

```diff
@@ -0,0 +1,213 @@
+---
+title: TiDB 6.5.4 Release Notes
+summary: 了解 TiDB 6.5.4 版本的兼容性变更、改进提升，以及错误修复。
+---
+
+# TiDB 6.5.4 Release Notes
+
+发版日期：2023 年 x 月 x 日
+
+TiDB 版本：6.5.4
+
+试用链接：[快速体验](https://docs.pingcap.com/zh/tidb/v6.5/quick-start-with-tidb) | [生产部署](https://docs.pingcap.com/zh/tidb/v6.5/production-deployment-using-tiup) | [下载离线包](https://cn.pingcap.com/product-community/?version=v6.5.4#version-list)
+
+## 改进提升
+
++ TiDB <!-- tw: Oreoxmt 1-->
+
+    - 新增 stale read traffice 相关指标和监控面板 [#43325](https://github.com/pingcap/tidb/issues/43325) @[you06](https://github.com/you06)
+    - (dup): release-7.3.0.md > 改进提升> TiDB - 优化与落盘相关的 chunk 读取的性能 [#45125](https://github.com/pingcap/tidb/issues/45125) @[YangKeao](https://github.com/YangKeao)
+
++ TiKV <!-- tw: Oreoxmt 9-->
+
+    - Add get_region_read_progress command in tikv-ctl [#15082](https://github.com/tikv/tikv/issues/15082) @[ekexium](https://github.com/ekexium)
+    - Compress traffic of check_leader requests. [#14839](https://github.com/tikv/tikv/issues/14839) @[you06](https://github.com/you06)
+    - rawkv: fix ttl_checker for RawKV API v2. [#15142](https://github.com/tikv/tikv/issues/15142) @[pingyu](https://github.com/pingyu)
+    - Make the checkpoint lag of PITR more stable when there are some leadership transforming. [#13638](https://github.com/tikv/tikv/issues/13638) @[YuJuncen](https://github.com/YuJuncen)
+    - Add safe-ts related logs and Grafana panels. [#15082](https://github.com/tikv/tikv/issues/15082) @[ekexium](https://github.com/ekexium)
+    - Fix occasional panic during shutdown [#13926](https://github.com/tikv/tikv/issues/13926) @[BusyJay](https://github.com/BusyJay)
+    - Make online unsafe recovery abort on timeout [#15346](https://github.com/tikv/tikv/issues/15346) @[Connor1996](https://github.com/Connor1996)
+    - 为 check leader 请求添加 gzip 压缩降低跨 tikv 节点之间相关流量开销 [#14553](https://github.com/tikv/tikv/issues/14553) @[you06](https://github.com/you06)
+    - 新增 min_safe_ts, min_safe_ts_region 和 min_safe_ts_gap 监控指标，用于诊断 resolved-ts 相关问题 [#15082](https://github.com/tikv/tikv/issues/15082) @[ekexium](https://github.com/ekexium)   
```

Comment:
```suggestion
    - (dup): 使用 gzip 压缩 `check_leader` 请求以减少流量 [#14553](https://github.com/tikv/tikv/issues/14553) @[you06](https://github.com/you06)
```
Remove `Fix occasional panic during shutdown [#13926](https://github.com/tikv/tikv/issues/13926) @[BusyJay](https://github.com/BusyJay)` as it was fixed in v6.5.1